neurolib/utils/functions.py

```diff
@@ -1,12 +1,13 @@
 import logging
 import numpy as np
 import scipy.signal
+from numba import jit
```

Comment:
Just for consistency: Please use the convention: 
`import numba` 
and decorate with 
`@numba.njit` instead of `@jit(nopython=True)`